src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -7,25 +7,46 @@ import snyk.amplitude.api.AmplitudeExperimentApiClient
 import snyk.amplitude.api.AmplitudeExperimentApiClient.Defaults.FALLBACK_VARIANT
 import snyk.amplitude.api.ExperimentUser
 import snyk.amplitude.api.Variant
+import java.io.IOException
+import java.util.Properties
 import java.util.concurrent.ConcurrentHashMap
 
+private val LOG = logger<AmplitudeExperimentService>()
+
 @Service
 class AmplitudeExperimentService : Disposable {
-    private val log = logger<AmplitudeExperimentService>()
 
-    private val apiClient: AmplitudeExperimentApiClient =
-        AmplitudeExperimentApiClient.create(apiKey = "client-bQAwhmzLhgpgqAhKAqRzIjnS915YrAqj")
+    private var amplitudeLoaded = false
+    private var apiClient: AmplitudeExperimentApiClient = AmplitudeExperimentApiClient.create(apiKey = "")
     private val storage: ConcurrentHashMap<String, Variant> = ConcurrentHashMap()
```

Comment:
rename to `key2variant` to have more context in name?

Reply:
as discussed in previous PR it's almost copy-paste from Amplitude Android SDK, as they will have Kotlin SDK this code will be as 3rd party library and not be presented by us.